.gitignore

```diff
@@ -1,5 +1,6 @@
 /node_modules
-package-lock.json
```

Comment:
I don't think this file is needed.

Reply:
That line is just moved around to alphabetic order. It has been added at the end of the file. My editor seems to like to order lines in `.gitignore`

Reply:
Oh, ok.